test/regression.py

```diff
@@ -614,6 +614,16 @@ def testJaguar_Jaguar8_3_stopiter_jaguar_hf_out(logfile):
 
 # Molcas #
 
+def testMolcas_Molcas8_0_dvb_sp_out(MolcasSPTest):
+ """Test to confirm that MOs are parsed."""
+ def test_for_mo_parsing(self):

```

Comment:
This defines a nested function, but doesn't actually do anything. Just do the assert in `testMolcas*`, like in other regression test functions.